src/svelte/Components/Macro.svelte

```diff
@@ -1,11 +1,14 @@
 <script>
-  import { editStore, removeMacro } from "../stores";
+  import { editStore, removeMacro, macroStore } from "../stores";
   import MacroActionButton from "./MacroActionButton.svelte";
 
   export let macro;
   export let idx;
+
+  $macroStore[idx].visibility = 'visible'; // every macro should be visible on startup
 </script>
 
+{#if $macroStore[idx].visibility === 'visible'}
```

Comment:
warum kein boolean? ist das nicht viel einfacher? dann könnte man das so schreiben: 
```suggestion
{#if $macroStore[idx].isVisible}
```